tests/unit-tests/test_config.py

```diff
@@ -2263,19 +2263,34 @@ def test_multi_task_config():
         assert efr_config.eval_metric[0] == "rmse"
         assert efr_config.batch_size == 64
 
-if __name__ == '__main__':
-    test_multi_task_config()
-    test_id_mapping_file()
-    test_load_basic_info()
-    test_gnn_info()
-    test_load_io_info()
-    test_train_info()
-    test_rgcn_info()
-    test_rgat_info()
-    test_node_class_info()
-    test_node_regress_info()
-    test_edge_class_info()
-    test_lp_info()
-
-    test_lm()
-    test_check_node_lm_config()
\ No newline at end of file
+
+def test_save_combined_config():
```

Comment:
One question on the test, as our yaml config is nested in ways like:

```
gnn:
   lr: 0.05
```

Do we keep the same nested structure as the original one? Or we can have some tests on it as well.

Reply:
If the runtime arg already existed in one of the YAML sections we just update its value. 

If the runtime argument did not already exist in the YAML, we decided to add it to its own section called `runtime` after discussion with @classicsong